pkg/kubelet/kubelet.go

```diff
@@ -601,144 +600,103 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	pluginSettings.LegacyRuntimeHost = nl
 
 	if containerRuntime == kubetypes.RktContainerRuntime {
-		glog.Warningln("rktnetes has been deprecated in favor of rktlet. Please see https://github.com/kubernetes-incubator/rktlet for more information.")
-	}
-
-	// rktnetes cannot be run with CRI.
-	if containerRuntime != kubetypes.RktContainerRuntime {
```

Comment:
I assume you simply remove the `if`? It's hard to tell from looking at the diff.

Reply:
It is much easier to see if you ignore whitespace

Reply:
Yes, indeed better asking GitHub to ignore whitespace when looking at the diff.

To answer your question: Yes, only the "if" was removed, and indentation adjusted.